core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1804,6 +1798,21 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
     }
   }
 
+  def getGenomicsOperation(workspaceName: WorkspaceName, jobId: String): Future[PerRequestMessage] = {
+    // First check the workspace context and permissions in a DB transaction.
+    // We don't need any DB information beyond that, so just return Unit on success.
+    dataSource.inTransaction { dataAccess =>
+      withWorkspaceContextAndPermissions(workspaceName, WorkspaceAccessLevels.Read, dataAccess) { _ =>
+        DBIO.successful(())
+      }
+    }
+    // Next call GenomicsService, which actually makes the Google Genomics API call.
+    .flatMap { _ =>
+      val genomicsServiceRef = context.actorOf(GenomicsService.props(genomicsServiceConstructor, userInfo))
+      (genomicsServiceRef ? GenomicsService.GetOperation(jobId)).mapTo[PerRequestMessage]
```

Comment:
Well this blog post is only a week or two old, althogh they've been pushing out sequels at a good clip. I've been taking an "I'll believe it when I see it" approach as typed akka has been Coming Soon (including here and gone already) for so many years, but it looks like they're serious this time.